[2024-08-29T16:28:37.039+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-29T16:28:37.047+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2024-08-29T16:28:31.848747+00:00 [queued]>
[2024-08-29T16:28:37.053+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2024-08-29T16:28:31.848747+00:00 [queued]>
[2024-08-29T16:28:37.053+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 3
[2024-08-29T16:28:37.071+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): tsk_load_to_s3> on 2024-08-29 16:28:31.848747+00:00
[2024-08-29T16:28:37.074+0000] {standard_task_runner.py:72} INFO - Started process 7631 to run task
[2024-08-29T16:28:37.079+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'zillow_analytics_dag', 'tsk_load_to_s3', 'manual__2024-08-29T16:28:31.848747+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/zillow_analytics.py', '--cfg-path', '/tmp/tmpx6nnv89a']
[2024-08-29T16:28:37.080+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask tsk_load_to_s3
[2024-08-29T16:28:37.124+0000] {task_command.py:467} INFO - Running <TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2024-08-29T16:28:31.848747+00:00 [running]> on host ip-172-31-11-44.eu-west-3.compute.internal
[2024-08-29T16:28:37.195+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='says@master.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='zillow_analytics_dag' AIRFLOW_CTX_TASK_ID='tsk_load_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-08-29T16:28:31.848747+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-29T16:28:31.848747+00:00'
[2024-08-29T16:28:37.195+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-29T16:28:37.209+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-29T16:28:37.210+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'aws s3 mv /home/ubuntu/response_data_29082024162833.json s3://endtoend-bucket001/']
[2024-08-29T16:28:37.216+0000] {subprocess.py:86} INFO - Output:
[2024-08-29T16:28:37.855+0000] {subprocess.py:93} INFO - Completed 55.8 KiB/55.8 KiB (286.2 KiB/s) with 1 file(s) remainingmove: ../../home/ubuntu/response_data_29082024162833.json to s3://endtoend-bucket001/response_data_29082024162833.json
[2024-08-29T16:28:37.931+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-29T16:28:37.954+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-29T16:28:37.954+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=zillow_analytics_dag, task_id=tsk_load_to_s3, run_id=manual__2024-08-29T16:28:31.848747+00:00, execution_date=20240829T162831, start_date=20240829T162837, end_date=20240829T162837
[2024-08-29T16:28:37.978+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-29T16:28:37.993+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-29T16:28:38.004+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
